packages/api-client/src/auth/parseAccessToken.test.ts

```diff
@@ -0,0 +1,41 @@
+import {AccessTokenType, parseAccessToken, parseValue} from './parseAccessToken';
```

Comment:
Missing copyright header

Reply:
Time to add: https://github.com/nikku/eslint-plugin-license-header 😎